circleci-orbs/preview.yml

```diff
@@ -0,0 +1,95 @@
+description: Easily register and deploy a component to a new environment on the Architect Cloud, then destroy the component and environment when ready
+
+display:
+  home_url: https://architect.io
+  source_url: https://github.com/architect-team/architect-cli/tree/master/circleci-orbs
+
+jobs:
+  create:
+    parameters:
+      email:
+        type: string
+      password:
+        type: string
+      account:
+        type: string
+      environment:
+        type: string
+      platform:
+        type: string
+      component_name:
+        type: string
+      component_file:
+        type: string
+        default: ./architect.yml
+      deploy_flags:
+        type: string
+        default: ''
+    executor: preview_executor
+    steps:
+      - run: npm install -g @architect-io/cli@0.7
```

Comment:
Can we parameterize the install? Auto publishing seems kind of mute if we are pinned to a specific version of our cli.

Reply:
Parameterize it as in change the version to be installed? What if the flags/args change?

Reply:
It would be queued off of the cli tag so if the flags changed we would have had to update the orb since its in the same repo. 

We could potentially add a test to test the orb with the current version of the cli before publishing.

Reply:
Lets talk about this in our quick meeting tomorrow